### PR TITLE
Basic mock data

### PR DESCRIPTION
**Pull request by davidguy3237**
I can't guarantee that this data will work.

SETUP INSTRUCTIONS:

1. login to psql
2. `delete database <database_name>`
3. `create database <database_name>`
4. logout of psql
5. `psql <database_name> -f <direct file path to blue-ocean-schema.sql>`
6. `psql <database_name> -f <direct file path to users-seed.sql>`
7. `psql <database_name> -f <direct file path to trades-seed.sql>`
8. `psql <database_name> -f <direct file path to ratings-seed.sql>`

The first 3 albums in the Rock category will have open trades you can browse
Specifically: Three Imaginary Boys, Greatest Hits, and The Synarchy of Molten Bones
no other album has any open trade data.
image

### CODE REVIEW

File: server/database/schemas/messages-seed.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO messages(sender_id, recipient_id, trade_id, body)
+VALUES('cliuo1dcs000208i9hga217k5', 'cliunrnur000008ms1u5qhzl2', 1, 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')
```
Comment (seguyy) [inline server/database/schemas/messages-seed.sql]: Hi @davidguy3237 I don't see created_at data insertion here for this messages table. Did it work for your test?

File: server/database/schemas/ratings-seed.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- Ratings For Completed Trades
+INSERT INTO ratings(sender_id, recipient_id, rating, trade_id)
```
Comment (seguyy) [inline server/database/schemas/ratings-seed.sql:2]: same, no created_at insertion

File: server/database/schemas/trades-seed.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
+-- Completed Trades
+INSERT INTO trades(seller_id, have_album_id, want_album_id, buyer_id, status, description)
```
Comment (seguyy) [inline server/database/schemas/trades-seed.sql:3]: same as above

File: server/database/schemas/users-seed.sql [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+VALUES('cliunrnur000008ms1u5qhzl2','fake1@email.com' 'fake_user1', 'https://i.pinimg.com/originals/bf/a7/8b/bfa78b30dbb524beb4104e471a1b6dd1.jpg', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.', 'Placeholder, Placeholder');
+
+INSERT INTO users(uid, email, username, profile_picture, , biography, location)
+VALUES('cliuo1dcs000208i9hga217k5','fake2@email.com' 'fake_user2', 'https://pbs.twimg.com/media/EvyB3s9WgAAROdH.jpg', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.', , 'Placeholder, Placeholder');
+
+INSERT INTO users(uid, email, username, profile_picture, , biography, location)
+VALUES('cliuo1p5o000308i99dba68wy','fake3@email.com' 'fake_user3', 'https://pbs.twimg.com/media/E0kJE8wWYAcWmgo.jpg', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.', , 'Placeholder, Placeholder');
+
+INSERT INTO users(uid, email, username, profile_picture, , biography, location)
```
Comment (seguyy) [inline server/database/schemas/users-seed.sql]: see my above comments

**Review by seguyy — APPROVED** [review]

**Review by Dmigdol — APPROVED** [review]

**Pull request merged**